recipes-core/cvm-reverse-proxy-server/files/cvm-reverse-proxy-server-init

```diff
@@ -0,0 +1,55 @@
+#!/bin/sh
+### BEGIN INIT INFO
+# Provides:          cvm-reverse-proxy-server
+# Required-Start:    $network $remote_fs $syslog
+# Required-Stop:     $network $remote_fs $syslog
+# Default-Start:     2 3 4 5
+# Default-Stop:      0 1 6
+# Short-Description: Start CVM Reverse Proxy Server at boot time
+# Description:       Enable CVM Reverse Proxy Server service.
+### END INIT INFO
+
+DAEMON=/usr/bin/proxy-server
+NAME=cvm-reverse-proxy-server
+DESC="CVM Reverse Proxy Server"
+DAEMON_ARGS="--listen-addr=localhost:7946 --target-addr=http://localhost:7936 --server-attestation-type azure-tdx"
```

Comment:
that one is for public access, proxying to orderflow-proxy?

if so, then it should:
1. listen on port 7936
2. proxy to localhost:14727

Reply:
Why have a separate cvm server proxy for the orderflow-proxy and not have them built in a single one directly?
are there any other purposes for the cvm-reverse-proxy-server beside forwarding to the orderflow proxy?

Reply:
- one receives a request from outside and forwards it to the local orderflow-proxy service
- the other receives a request from orderflow-proxy or rbuilder, and forwards it to BuilderHub

Reply:
I meant only about the orderflow proxy and cvm-reverse-proxy server. Why separate them instead of have them both in one?